server/runtime/src/main/java/org/infinispan/server/state/ServerStateManagerImpl.java

```diff
@@ -0,0 +1,290 @@
+         return cache.putAsync(IGNORED_CACHES_KEY, ignored).thenApply(r -> null);
+      }
+   }
+
+   @Override
+   public CompletableFuture<Void> ignoreCache(String cacheName) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      synchronized (this) {
+         ignored.caches.add(cacheName);
+         hasIgnores = !ignored.caches.isEmpty();
+         return cache.putAsync(IGNORED_CACHES_KEY, ignored).thenApply(r -> null);
+      }
+   }
+
+   @Override
+   public Set<String> getIgnoredCaches() {
+      return Collections.unmodifiableSet(ignored.caches);
+   }
+
+   @Override
+   public boolean isCacheIgnored(String cacheName) {
+      return hasIgnores && ignored.caches.contains(cacheName);
+   }
+
+   @Override
+   public CompletableFuture<Boolean> connectorStart(String name) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      return cache.removeAsync(new ScopedState(CONNECTOR_STATE_SCOPE, name), true).thenApply(v -> null);
+   }
+
+   @Override
+   public CompletableFuture<Void> connectorStop(String name) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      return cache.putAsync(new ScopedState(CONNECTOR_STATE_SCOPE, name), true).thenApply(v -> null);
+   }
+
+   @Override
+   public CompletableFuture<Boolean> connectorStatus(String name) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      return CompletableFuture.completedFuture(server.getProtocolServers().get(name).getTransport().isRunning());
+   }
+
+   @Override
+   public CompletableFuture<Void> setConnectorIpFilterRule(String name, Collection<IpSubnetFilterRule> filterRule) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      IpFilterRules ipFilterRules = new IpFilterRules();
+      filterRule.forEach(r -> ipFilterRules.rules.add(new IpFilterRule(r)));
+      return cache.putAsync(new ScopedState(CONNECTOR_IPFILTER_SCOPE, name), ipFilterRules).thenApply(v -> null);
+   }
+
+   @Override
+   public CompletableFuture<Void> clearConnectorIpFilterRules(String name) {
+      SecurityActions.checkPermission(cacheManager, AuthorizationPermission.ADMIN);
+      return cache.putAsync(new ScopedState(CONNECTOR_IPFILTER_SCOPE, name), new IpFilterRules()).thenApply(v -> null);
+   }
+
+   private void updateLocalIgnoredCaches(IgnoredCaches ignored) {
+      if (ignored != null) {
+         synchronized (this) {
+            this.ignored.caches.clear();
+            this.ignored.caches.addAll(ignored.caches);
+            hasIgnores = !this.ignored.caches.isEmpty();
+         }
+      }
+   }
+
+   private CompletionStage<Void> updateIpFilters(String connector, Collection<IpFilterRule> rules) {
+      ProtocolServer protocolServer = server.getProtocolServers().get(connector);
+      List<IpSubnetFilterRule> localRules = new ArrayList<>(rules.size());
```

Comment:
can be simplified for the case where `rules.isEmpty()`: (I think)
* set an empty list `protocolServer.getConfiguration().ipFilter().rules(Collections.emptyList());`
* log it: `log.connectorIpFilterCleared()`